backend.py

```diff
@@ -60,6 +54,35 @@ def __repr__(self):
         return "<State> {} {}>".format(self.board, self.robots)
 
 
+class Rotation(Enum):
+    N = 0
+    E = 90
+    S = 180
+    W = 270
+
+    def __add__(self, other):
+        return Rotation(self.value + other.value)
```

Comment:
Nemá tam být taky modulo 360, jak je dole?

Reply:
Cau: melo a nemelo zaroven. Zjistila jsem, ze metodu __add__ neni nutne prepisovat, protoze class.value, jako typ int, se v pohode scita s dalsimi cisly. To byl pohrobek prvni implementace, kde jsem chtela testove scitat Rotation.N + Rotation.E (coz je pekna blbost :)).

Reply:
Aha, dobře, díky ;)